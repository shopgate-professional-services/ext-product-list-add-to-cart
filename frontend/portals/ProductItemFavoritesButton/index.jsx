import React from 'react';
import CTAButtons from './components/CTAButtons';

/**
 * Renders CTA buttons in product-item.favorites-button portal
 * @param {Object} props props
 * @returns {JSX}
 */
const ProductItemFavoritesButton = (props) => {
  return (<CTAButtons {...props} />);
};

export default ProductItemFavoritesButton;
