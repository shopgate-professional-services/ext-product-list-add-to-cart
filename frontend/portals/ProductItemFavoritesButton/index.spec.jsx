import React from 'react';
import { shallow } from 'enzyme';
import ProductItemFavoritesButton from './index';

jest.mock('./components/CTAButtons', () => <div>CTAButtons</div>);

describe('ProductItemFavoritesButton', () => {
  it('should render', () => {
    const component = shallow(<ProductItemFavoritesButton />);
    expect(component).toMatchSnapshot();
  });
});
