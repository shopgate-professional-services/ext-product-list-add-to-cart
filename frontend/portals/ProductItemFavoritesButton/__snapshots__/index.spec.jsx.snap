// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductItemFavoritesButton should render 1`] = `<[object Object] />`;
